recipes/x86_64_level/meta.yaml

```diff
@@ -0,0 +1,68 @@
+package:
+  name: x86_64-microarch-level-split
+  version: {{ level }}
+
+build:
+  number: 0
+  string: 0_{{ microarchitecture }}
```

Comment:
Do we want the build number at the end?

Reply:
I usually have the feature at the end. for eg: `libblas=*=*_mkl`.

Reply:
```suggestion
  string: {{ number }}_{{ microarchitecture }}
```